src/db/factories/users.factory.ts

```diff
@@ -0,0 +1,16 @@
+import { Faker } from '@faker-js/faker';
+import { setSeederFactory } from 'typeorm-extension';
+import { User } from '../entities/user.entity';
+
+export const UsersFactory = setSeederFactory(User, (faker: Faker): User => {
+  const user = new User();
```

Comment:
Maybe let's seed our database with the same values that we had in db.json? (4 users)